src/control_protocol.rs

```diff
@@ -61,6 +75,8 @@ pub async fn main_control(
 pub trait ControlProtocol {
     async fn send_message(&self, args: SendMessageArgs) -> Result<(), SendMessageError>;
 
+    async fn send_global_rpc(&self, args: SendGlobalRpcArgs) -> Result<(), SendGlobalRpcError>;
```

Comment:
This should return a serde_json::Value. Otherwise we cannot see what the RPC call returns.